Modules/ChequeModule/Models/ChequeModel.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ChequeWriter.Modules.ChequeModule.Models
+{
+    public class ChequeModel
+    {
+        public int Id { get; set; }
+
+        public string Name { get; set; }
+
+        public double Amount { get; set; }
+
+        public DateTime DateCreated { get; set; }
+
+        public string AmountInWord { get; set; }
+
+        public ChequeModel(string name, string amtInWord)
```

Comment:
The proper way to create a Cheque object will need to pass thru the constructor (Line 21). By using amount in word as parameter to create a cheque object might not a proper way, We should use `double` variable type instead of `string`. Can you change the passing parameter to `double` ?